R/model-module-helpers.R

```diff
@@ -217,7 +217,7 @@ add_pmfs <- function(pmfs) {
 #' mat <- matrix(1:9, nrow = 3)
 #' extract_sparse_matrix(mat)
 extract_sparse_matrix <- function(mat, prefix = "") {
-  sparse_mat <- rstan::extract_sparse_parts(mat)
+  sparse_mat <- suppressMessages(rstan::extract_sparse_parts(mat))
```

Comment:
@seabbs We could do this, but if we copied over code from rstan we would have a license issue since they use GPL, and we use MIT currently. So we should either code something similar on our own or adapt our license, which would make the package more restrictive, however.